server/data/mappers/educationMapper.ts

```diff
@@ -1,8 +1,37 @@
-import type { EducationResponse } from 'educationAndWorkPlanApiClient'
-import type { EducationDto } from 'inductionDto'
+import { parseISO } from 'date-fns'
+import type { AchievedQualificationResponse, EducationResponse } from 'educationAndWorkPlanApiClient'
+import type { EducationDto, QualificationDto } from 'dto'
+import EducationLevelValue from '../../enums/educationLevelValue'
+import QualificationLevelValue from '../../enums/qualificationLevelValue'
 
-function toEducationResponse(educationDto: EducationDto): EducationResponse {
-  return { ...educationDto }
+const toEducationDto = (educationResponse: EducationResponse, prisonNumber: string): EducationDto => {
```

Comment:
The mapper function had the wrong name and signature - it needs to take in an EducationResponse (ie. the raw API response type) and map it into an EducationDto

The mapping itself is also more complex that a simple field by field copy as we need to convert dates (the raw API returns them as strings, but we need to parse them into Date objects); and also the same for the EducationLevelValue and QualificationLevelValue enums